### DISCUSSION TITLE
how to view tweet with replies

### DISCUSSION THREAD

**Discussion by ghost** [jonjomckay/fritter#300] [Q&A] [answered]
I'm not able to do so (Galaxy Note 10+ 5g)

**Comment by ghost** [comment]
Seems mile the f-droid Version is very outdated (2.2.0)

**Reply by jonjomckay** [reply]
The F-Droid version should be completely up to date (v2.15.1). Please check to make sure your F-Droid client is updating its repositories correctly!

**Reply by ghost** [reply]
Weird, now it's 2.15.1 in F-Droid (however i'm not sure if it just shows the installed version from github now). Shouldn't F-Droid also list old versions (i only see one version and it has been the same for 2.2.0 before)?

**Comment by jonjomckay** [comment, marked as answer]
This is supported since [v2.11.0](https://github.com/jonjomckay/fritter/releases/tag/v2.11.0)! Please ensure your app is up-to-date :)
